src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyRulesForm.tsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import omit from 'lodash/omit';
+
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import SelectControl from 'app/components/forms/selectControl';
+import TextField from 'app/components/forms/textField';
```

Comment:
`-Fields` are generally used with the accompanying `<Form>` component which provides some magic. `-Control` components are generally the form inputs that don't have the Form logic, and just provide a more presentational form input. We can use `app/views/settings/components/forms/input.jsx` (it's in a bad directory, and needs to replace app/components/forms in the future).

Reply:
Honestly, I tried to use our Component form to be consistent, but it didn't meet my needs (note that we're adding fields dynamically) ... so, after playing around with the Form component and with it's onChange and WillComponentUpdate methods, I decided not use it and leave the code as it is, otherwise I would have to change many things, risking breaking something in the application ... this is also a component that deserves special attention if we decide on a refactor